### DISCUSSION TITLE
[Feature Request] - Permanent NFC activation

### DISCUSSION THREAD

**Discussion by titusz** [btcpayserver/btcpayserver#4808] [Ideas / Feature Requests]
Currently using the Keypad POS, the merchant has to manually activate NFC scanning at every checkout. It would be great if the user could enable NFC reading permanently, maybe by offering a toggle switch for activating NFC instead of a button.

While activating NFC in the browser does require an active user action to give permission, this is only required the first time and should usually persist across browser session. The app could check if the permission is available and activate automatically if the toggle is on. See: https://web.dev/nfc/#check-for-permission

**Comment by Zaxounette** [comment]
@dennisreimann Should have the "always on" NFC activation good to go shortly. Actual ETA unknown.
He posted a youtube video to explain the issue at hand (exactly what you stated), and the resolution.

https://www.youtube.com/watch?v=CvckVuSwxis

**Reply by dennisreimann** [reply]
Yup, this is solved with PRs that got merged yesterday. Will be part of v1.9 soon.
